scripts/release.js

```diff
@@ -1,211 +1,168 @@
-}
+  //Bump next development version
+  versionBump();
+};
 
-// Checkout the meeting demo
-process.chdir(path.join(__dirname, '../..'));
-if (!fs.existsSync('amazon-chime-sdk')) {
-  spawnOrFail('git', ['clone --branch meeting-dev https://github.com/aws-samples/amazon-chime-sdk.git']);
-} else {
-  spawnOrFail('rm', ['-rf amazon-chime-sdk']);
-  spawnOrFail('git', ['clone --branch meeting-dev https://github.com/aws-samples/amazon-chime-sdk.git']);
-}
+const hotfix = () => {
+  if (isPreRelease(currentVersion)) {
+    logger.error(`We currently do not do hotfix for pre-release version.`);
+    process.exit(1);
+  }
+  cleanUp();
+  const newVersion = versionBump(1, 'hotfix');
+  updateJSSdkDep();
+  buildAndPack();
+  checkNPMDepsInstall();
+  deployDemo(newVersion);
+};
+
+const main = () => {
+  //  Go to root dir
+  process.chdir(path.join(__dirname, '..'));
 
-// Udpate the demo to the most up to date version of JS SDK 
-process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting'));
-updateDependency(`amazon-chime-sdk-js@${updatedSdkVersion}`);
-// Udpate the demo to the most up to date version of React library
-process.chdir(path.join(__dirname, '..'));
-packDependency();
-process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting'));
-updateDependency('amazon-chime-sdk-component-library-react', `../../../amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-${versionString}.tgz`);
-
-process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting/serverless'));
-logger.log('Deploying unique release candidate Meeting Demo URL...');
-const formattedVersion = versionString.replace(/\./g, '-');
-spawnOrFail('node', [`./deploy.js -r us-east-1 -b chime-sdk-components-demo-${formattedVersion} -s chime-sdk-components-demo-${formattedVersion}`]);
-
-process.chdir(path.join(__dirname, '..'));
-spawnOrFail('rm', [`-rf amazon-chime-sdk-component-library-react-${versionString}.tgz`]);
-process.chdir(path.join(__dirname, '../..'));
-spawnOrFail('rm', ['-rf amazon-chime-sdk']);
\ No newline at end of file
+  logger.log('Choose one of the following options:');
+  logger.log('  1. Release');
+  logger.log('  2. Hotfix');
+  logger.log('  3. Deploy demo');
+  const option = prompt('');
+
+  switch (option) {
+    case '1':
+      release();
+      break;
+    case '2':
+      hotfix();
+      break;
+    case '3':
+      const remoteBranch = getCurrentRemoteBranch();
+      if (!remoteBranch || (!(remoteBranch.includes('origin/release-')) && (remoteBranch !== 'origin/hotfix'))) {
+        logger.error(`The local branch ${currentBranch} does not track either release or hotfix branch`);
+        process.exit(1);
+      }
+      cleanUp(remoteBranch);
+      spawnOrFail('npm', ['install']);
+      deployDemo(currentVersion);
+      break;
+    default:
+      if (option) {
+        logger.error('Invalid option');
+      }
+      process.exit(1);
+  }
```

Comment:
Can you remidn me again why we need this? When would we use it?

Reply:
Sorry need what?

Reply:
why do we need the 'deploy demo' option

Reply:
So just when we need to include new commits in the release or QA finds bugs and we need to include fix, we will do cherry-pick those commits and just redeploy and skip all the previous steps of creating a release branch and bumping new dev versions.